src/server_manager/messages/en.json

```diff
@@ -40,15 +40,20 @@
   "error-key-remove": "Failed to remove key",
   "error-key-rename": "Failed to rename key",
   "error-keys-get": "Could not load keys",
+  "error-keys-port-bad-input": "The port must be an integer between 1 and 65,535.",
+  "error-keys-port-in-use": "The port is already in use on the server.",
+  "error-keys-port-not-saved": "Not Saved",
```

Comment:
We can make this key more general, i.e. `error-not-saved`

Reply:
Done